README.md

```diff
@@ -78,6 +78,10 @@ There are many more parameters you can use. Display the usage doc with the follo
 
     GitLink.exe -h
 
+### Native PDBs
+
+Native PDBs (from C++ projects) are supported by using -a option. All .cpp/.c/.h files from your git depot will be indexed in the PDB.
```

Comment:
Please add a usage example so people don't have to guess that it's just a flag and doesn't require an additional value.